projects/ccf-rui/src/app/modules/left-sidebar/left-sidebar.component.html

```diff
@@ -10,4 +10,24 @@
     <ccf-labeled-slide-toggle class="toggle side" [labels]="['L', 'R']"></ccf-labeled-slide-toggle>
     <ccf-labeled-slide-toggle class="toggle gender" [labels]="['Male', 'Female']"></ccf-labeled-slide-toggle>
   </div>
+  <hr>
+  <mat-expansion-panel class="extraction-menu">
```

Comment:
Expansion panels are briefly open before being hidden again, which causes a flash of content. Address this by having the expansion panels closed by default at the beginning.